app/constants/urls.js

```diff
@@ -1,24 +1,58 @@
 import ENV from 'website-www/config/environment';
 
-export const APPS = {
-  HOME: ENV.APPS?.HOME,
-  WELCOME: ENV.APPS?.WELCOME,
-  EVENTS: ENV.APPS?.EVENTS,
-  MEMBERS: ENV.APPS?.MEMBERS,
-  STATUS: ENV.APPS?.STATUS,
-  PROFILE: ENV.APPS?.PROFILE,
-  LIVE: 'live',
+const SCHEMA = 'https://';
```

Comment:
Isn't the name `SCHEMA` seems confusing?
Can we name it `PROTOCOL`?

Reply:
https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#scheme

Reply:
I can change it its confusing

Reply:
Hmm got it👍, we can keep it